Makefile

```diff
@@ -119,12 +119,16 @@ lint: golangci-bin ## Run configured golangci-lint and pre-commit.sh linters aga
 	hack/pre-commit.sh
 
 .PHONY: create-rdr-env
-create-rdr-env: ## Create a new rdr environment.
+create-rdr-env: drenv-prereqs ## Create a new rdr environment.
 	./hack/dev-env.sh create
 
-destroy-rdr-env: ## Destroy the existing rdr environment.
+destroy-rdr-env: drenv-prereqs ## Destroy the existing rdr environment.
 	./hack/dev-env.sh destroy
 
+.PHONY: drenv-prereqs
+drenv-prereqs: ## Check the prerequisites for the drenv tool.
+	./hack/ensure-drenv-prereqs.sh
```

Comment:
This script does not ensure the tools are there, so maybe "check-drenv-prereqs.sh"?

Reply:
Done